spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -428,12 +428,36 @@ def when_I_unschedule_publishing
   end
 
   def and_the_step_by_step_is_not_editable
+    when_I_visit_the_secondary_content_page
+    then_I_can_see_the_existing_secondary_links
+    and_I_cannot_add_secondary_content_link
+    and_I_cannot_delete_secondary_content_links
+
     when_I_visit_the_publish_or_delete_page
     there_should_be_no_publish_button
     there_should_be_no_disgard_changes_button
     there_should_be_no_unpublish_button
   end
 
+  def when_I_visit_the_secondary_content_page
+    visit step_by_step_page_secondary_content_links_path(@step_by_step_page)
+  end
+
+  def then_I_can_see_the_existing_secondary_links
+    expect(find('tbody')).to have_content(@step_by_step_page.secondary_content_links.first.title)
+  end
+
+  def and_I_cannot_add_secondary_content_link
+    expect(page).to_not have_css("h2", text: "Add new secondary link")
+    expect(page).to_not have_css("button", text: "Add secondary link")
+  end
+
+  def and_I_cannot_delete_secondary_content_links
+    within("tbody") do
```

Comment:
I'd be tempted to broaden the search by removing the `within` line above ^

This is because if the interface changes in the future (i.e. for the button to be outside of `tbody`), and then we break the code that locks down the page, our tests would continue to pass and we wouldn't know about it.

Reply:
It should be possible to traverse each row in the table and check that it doesn't have a "delete" button.

Reply:
`within("tbody")` will already catch any "delete" buttons in table rows, as the table rows are inside "tbody". 👍 

Have thought about this a bit more now and this is fine - [`within` will raise an error](https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FSession:within) if the scope cannot be found (i.e. if we move away from using tables as markup), so this test shouldn't falsely pass.